linode/image/resource.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	LinodeImageCreateTimeout = 20 * time.Minute
+	LinodeImageCreateTimeout = 25 * time.Minute
```

Comment:
```suggestion
	LinodeImageCreateTimeout = 30 * time.Minute
```

Let's be a little bit more progressive on this one